subliminal/subtitle.py

```diff
@@ -232,8 +232,10 @@ def guess_matches(video, guess, partial=False):
     if video.resolution and 'screen_size' in guess and guess['screen_size'] == video.resolution:
         matches.add('resolution')
     # format
-    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
-        matches.add('format')
+    if video.format and 'format' in guess:
+        guess_format = guess['format'] if isinstance(guess['format'], list) else [guess['format']]
+        if any(gf.lower() == video.format.lower() for gf in guess_format):
```

Comment:
the problem with this logic is that video returns a wrong format it would give a false positive match.

Reply:
So if I understand correctly, if `guess['format']` contains a single format that does not match `video.format`, we should not call `matches.add('format')`? So basically, replace `any` with `all`? Might be misunderstanding.

Reply:
@sprt No. Look at my possible fix. But there is a lot of discussion on how to fix it:
https://github.com/Diaoul/subliminal/pull/836